packages/react-server/src/ReactFizzServer.js

```diff
@@ -1164,7 +1236,21 @@ function renderNodeDestructive(
         const lazyNode: LazyComponentType<any, any> = (node: any);
         const payload = lazyNode._payload;
         const init = lazyNode._init;
-        const resolvedNode = init(payload);
+        let resolvedNode;
+        if (__DEV__) {
+          // If a lazy element throws a componentStack frame will be popped but since
+          // it is an element and not a component it never added a frame. We
+          // add a fake frame here in case we throw and then remove it if we didn't
+          // @TODO decide if this frame showing up in dev component stacks is good
+          // Currently if a lazy element rejects the component stack will include this
+          // special frame. My current intuition is this is good and will aid in identifying
+          // where errors are happening.
+          pushBuiltInComponentStackInDEV(task, '~lazy-element~');
```

Comment:
I believe Fiber calls this `'Lazy'`.

https://github.com/facebook/react/blob/c16b005f2d71acde26658f035ff491a5e027da39/packages/react-reconciler/src/ReactFiberComponentStack.js#L39-L40

This Fiber only exists temporarily before we know what the component will be and then it changes tag.

Which is similar to this case.

So I think it makes sense to keep it but call it `'Lazy'` instead for parity.

Reply:
Yeah maybe I was being pedantic in thinking that distinguishing between Component and elements that are lazy would matter but in reflection it probably would make more sense to the average engineer that any React.Lazy shows up as a Lazy in the stack even if technically it's not in Component position

Reply:
Oh, then I misunderstood. I thought this was in the type position.

In that case having no stack would align more with the client.

Reply:
yeah that make sense